index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Landify Project</title>
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <header class="header">
+    <nav>
+     <ul>
+       <li>
+          <a href="http://lokum.tech" target="_blank" rel="noopener noreferrer"> <img src="./images/Darklogo.png" alt="logo of a company">
+         </a>
+         </li>
+       <li>
+         <a href="#">About</a>
+       </li>
+       <li>
+         <a href="#">Products</a>
+       </li>
+       <li>
+         <a href="#"> Pricing</a>
+       </li>
+       <li>
+         <a href="#"> Blog</a>
+       </li>
+       <li>
+         <a href="#"> Jobs</a>
+       </li>
+       <li>
+         <label for="">
+           <select aria-label="label for the select">
+              <option value="1"> More</option>
+              <option value="2"> Something</option>
+              <option value="3"> Nobody Knows</option>
+              <option value="4">Unknown</option>
+           </select>
+         </label>
+       </li>
+     </ul>
+     <ul>
+       <li>
+         <a href="#"><img src="./images/Google Play Badge.png" alt="Google Badge">
+         </a>
+       </li>
+       <li>
+         <a href="#">
+            <img src="./images/App Store Badge.png" alt="Apple store Badge">
+         </a>
+        </li>
+     </ul>
+    </nav>
+    <div class="down">
+     <span>
+       <h1>
+         Organise projects. <br>
+         Get more done.
+       </h1>
+       <button>
+         Get Started
+       </button>
+     </span>
+     <img src="./images/Phone Mockup.png" alt="First photo Mockup">
+    </div>
+ </header>
```

Comment:
same here
- Give class names with meaning
- From what we discussed in the meeting. think about structure first and use CSS to enhance

Reply:
i did, please can you check them one by one again if its considerable